### PR TITLE
feat: adjust WAL purge default configurations

### LINKED ISSUE #5106: Consider adjusting the default purge size and interval of the WAL
### What type of enhancement is this?

User experience

### What does the enhancement do?

The default purge config for the WAL may be too large
https://github.com/GreptimeTeam/greptimedb/blob/c732016fa0f6ebee536e00fd945a728ae79ce283/src/common/wal/src/config/raft_engine.rs#L53-L54

The WAL can easily consume more than 4GB of disk space. We may consider adjusting the default config.

### Implementation challenges

We may find a proper default value. Users can adjust the value if the default config becomes the bottleneck.

### PR DESCRIPTION
**Pull request by killme2008**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## Refer to a related PR or issue link (optional)

Close #5106

## What's changed and what's your intention?

Adjust the default value of WAL purge configuration.

## Checklist

- [ ] I have written the necessary rustdoc comments.
- [ ] I have added the necessary unit tests and integration tests.
- [ ] This PR requires documentation updates.

### CODE REVIEW

File: config/config.md [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
 | `mode` | String | `standalone` | The running mode of the datanode. It can be `standalone` or `distributed`. |
-| `enable_telemetry` | Bool | `true` | Enable telemetry to collect anonymous usage data. |
 | `default_timezone` | String | Unset | The default timezone of the server. |
 | `init_regions_in_background` | Bool | `false` | Initialize all regions in the background during the startup.<br/>By default, it provides services after all regions have been initialized. |
 | `init_regions_parallelism` | Integer | `16` | Parallelism of initializing regions. |
 | `max_concurrent_queries` | Integer | `0` | The maximum current queries allowed to be executed. Zero means unlimited. |
 | `runtime` | -- | -- | The runtime options. |
 | `runtime.global_rt_size` | Integer | `8` | The number of threads to execute the runtime for global read operations. |
 | `runtime.compact_rt_size` | Integer | `4` | The number of threads to execute the runtime for global write operations. |
+| `runtime.enable_telemetry` | Bool | `true` | Enable telemetry to collect anonymous usage data. Enabled by default. |
```
Comment (evenyag) [inline config/config.md:23]: @zyy17 The `enable_telemetry` flag isn't a runtime option. Looks like a bug in toml2docs.

Comment (killme2008) [reply config/config.md:23]: Let me move it up.

**Review by shuiyisong — APPROVED** [review]

File: src/common/wal/src/config/raft_engine.rs [hunk 2]
```diff
@@ -50,8 +50,8 @@ impl Default for RaftEngineConfig {
         Self {
             dir: None,
             file_size: ReadableSize::mb(256),
```
[the remark this answers is not in the document]
Comment (killme2008) [reply src/common/wal/src/config/raft_engine.rs:52]: I think so.

**Comment by evenyag** [comment]
> > We may need to run TSBS to see if the default options can cause regression
> 
> Can we quickly benchmark writing based on this branch?

Before
```
Summary:
loaded 1036800000 metrics in 278.533sec with 6 workers (mean rate 3722357.59 metrics/sec)
loaded 103680000 rows in 278.533sec with 6 workers (mean rate 372235.76 rows/sec)

Summary:
loaded 1036800000 metrics in 276.704sec with 6 workers (mean rate 3746967.42 metrics/sec)
loaded 103680000 rows in 276.704sec with 6 workers (mean rate 374696.74 rows/sec)
```

After
```
Summary:
loaded 1036800000 metrics in 281.590sec with 6 workers (mean rate 3681954.22 metrics/sec)
loaded 103680000 rows in 281.590sec with 6 workers (mean rate 368195.42 rows/sec)


Summary:
loaded 1036800000 metrics in 278.871sec with 6 workers (mean rate 3717847.60 metrics/sec)
loaded 103680000 rows in 278.871sec with 6 workers (mean rate 371784.76 rows/sec)


Summary:
loaded 1036800000 metrics in 275.096sec with 6 workers (mean rate 3768864.60 metrics/sec)
loaded 103680000 rows in 275.096sec with 6 workers (mean rate 376886.46 rows/sec)
```

**Pull request merged**